### DISCUSSION TITLE
secure your wordpress website

### DISCUSSION THREAD

**Discussion by Jaures-beinjamin** [community/community#35705] [Other Feature Feedback, Questions, & Ideas]
I would like to know how I can make my wordpress site more secure because I receive threats from hackers by email since my site is ranked 1st on google and my SEO is successful

**Comment by tinaponting** [comment]
https://github.com/tinaponting/protect-wordpress   and use what you want:)  look at: maybe txt.

**Comment by CharlesRakesh** [comment]
https://github.com/CharlesRakesh/Elementor-Pro-Form/issues/1
